packages/@aws-cdk/aws-ecs/README.md

```diff
@@ -1147,13 +1147,16 @@ Default Capacity Provider Strategy can be added by using the method addDefaultCa
 ```ts
 declare const capacityProvider: ecs.CapacityProvider;
 
-new ecs.Cluster(stack, 'EC2CPCluster', {
-  vpc,
-  defaultCapacityProviderStrategy: [
-    { capacityProvider: capacityProvider.capacityProviderName, base: 1, weight: 1 },
-    { capacityProvider: 'FARGATE_SPOT', weight: 1 },
-  ],
+const cluster = new ecs.Cluster(stack, 'EcsCluster', {
+  enableFargateCapacityProviders: true,
 });
+cluster.addAsgCapacityProvider(capacityProvider);
+
+cluster.addDefaultCapacityProviderStrategy([
+  { capacityProvider: 'FARGATE', base: 10, weight: 50 },
+  { capacityProvider: 'FARGATE_SPOT' },
```

Comment:
Don't you need to specify a weight for these?

Reply:
Actually for the second its an option to specify weight, but we cannot specify base for that.

Reply:
Okay, but this example will default the weight of `FARGATE_SPOT` to 0, which is not a realistic example. Why would someone specify it unless they wanted some tasks to launch in spot capacity? Please add a weight to FARGATE_SPOT.

Reply:
Ok sure